spec/actors/generic_files/actor_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
```

Comment:
I think we can remove this file, it appears to be a duplicate of https://github.com/osulp/Scholars-Archive/blob/feature/FinishedEmbargo/spec/actors/scholars_archive/generic_files/actor_spec.rb